eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl

```diff
@@ -22,6 +22,12 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 -->
+<!--
```

Comment:
@andreoss we don't need this. The object `org.eolang.rust` will do everything internally. No language support is required. It will just be a normal object, which will find the right `.so` file in classpath and call it using JNI.

Reply:
@yegor256 Done